packages/styles/scss/patterns/simplelongform/_simplelongform.scss

```diff
@@ -0,0 +1,119 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@mixin simplelongform { 
+  .#{$prefix}--simplelongform {
+    color: $text-01;
+    background: $ui-background;
+    &__container {
+      padding-left: $layout-01;
+      padding-right: $layout-01;
+    }
+    &__row {
+      @include carbon--make-row;
+    }
+    &__col {
+      @include carbon--make-col-ready;
+      @include carbon--make-col-offset(4, 16);
+      @include carbon--make-col(7, 16);
```

Comment:
This is your smallest breakpoint. There shouldn't be more than 4 columns here. Based on the specs it looks like this should be `(4, 4)`, and the offset isn't necessary.